lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -29,7 +29,7 @@ def source_dir
 
 def fixtures(category)
   begin
-    fixtures = YAML.load_file(".fixtures.yml")["fixtures"]
+    fixtures = YAML.load_file( ENV['FIXTURES_YML'] || '.fixtures.yml' )['fixtures']
```

Comment:
I get that. But, considering that these are minor changes and most people have Gemfile.lock in place when testing, what is the harm?

We'll all yell if it's broken ;-)